src/App.js

```diff
@@ -1,35 +1,31 @@
 import React from 'react';
-import BookCard from './bookCard/BookCard';
-import FormName from './FormName';
-import Header from './template/Header';
-import Footer from './template/Footer';
-import SimilarBookList from "./SimilarBookList";
+import Book from './components/pages/bookCard';
+import BookList from './components/pages/bookList';
+import NewBook from './components/pages/newBook';
+import EditBook from './components/pages/editBook';
+import Main from './components/pages/main';
+import Error404 from "./components/pages/404";
+import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
+import { newBookPath, editBookPath, bookPath, mainPath, booklistPath, contactPath } from "./services/helpers/routes";
+import Contacts from './Components/Pages/Contacts';
```

Comment:
кейс символов имеет значение, при сборке в некоторых операционных системах (например Windows) сборщик может выдать ошибку, так как название файлов при строгой проверке кейса не будет соответствовать

должно быть
```js
import Contacts from './components/pages/contacts';
```

Сейчас сборщик резонно выдает предупреждение
```
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior 
when compiling on a filesystem with other case-semantic.
```